modin/pandas/test/internals/test_benchmark_mode.py

```diff
@@ -23,14 +23,14 @@
 # We have to explicitly mock subclass implementations of wait_partitions.
 if engine == "Ray":
     wait_method = (
-        "modin.core.execution.ray.implementations.pandas_on_ray."
-        + "partitioning.partition_manager."
+        "modin.core.execution.ray.implementations."
+        + "pandas_on_ray.partitioning."
```

Comment:
Let's make this change in a separate PR where we define public interface for ray engine (e.g., expose the partition classes).

Reply:
For ray, the interface is already defined, I apparently missed this place last time.

Reply:
Should I put this change in a separate pull request anyway?

Reply:
I think so. In that PR we will also expose the classes from virtual_partition.py file.

Reply:
ok